src/main/java/com/epam/architecture/RESTws/UserRESTService.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.architecture.RESTws;
+
+import com.epam.architecture.roles.AuthorizationUtil;
+import com.epam.architecture.userinterface.LibraryService;
+import jakarta.annotation.Resource;
+import jakarta.xml.ws.WebServiceContext;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.Response;
+
+@Path("/user")
```

Comment:
Why user ?

Reply:
Because all actions with bookmarks are tied to a specific user